requirements/dev.txt

```diff
@@ -1,67 +1,201 @@
 # To update, run:
 #
 #    pip-compile --output-file=requirements/dev.txt requirements/dev.in
 #
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos  # via -r requirements/dev.in
--e .
--e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow  # via -r requirements/dev.in, kytos
-appdirs==1.4.4            # via virtualenv
-astroid==2.3.3            # via pylint
-backcall==0.1.0           # via ipython, kytos
-click==7.1.1              # via flask, kytos, pip-tools
-coverage==5.0.4           # via kytos-of-core
-decorator==4.4.2          # via ipython, kytos, traitlets
-distlib==0.3.1            # via virtualenv
-docopt==0.6.2             # via yala
-docutils==0.17            # via kytos, python-daemon
-filelock==3.0.12          # via tox, virtualenv
-flask-cors==3.0.8         # via kytos
-flask-socketio==4.2.1     # via kytos
-flask==1.1.2              # via flask-cors, flask-socketio, kytos
-importlib-metadata==1.7.0  # via pluggy, tox, virtualenv
-importlib-resources==5.1.2  # via virtualenv
-ipython-genutils==0.2.0   # via kytos, traitlets
-ipython==7.13.0           # via kytos
-isort==4.3.21              # via pylint, yala
-itsdangerous==1.1.0       # via flask, kytos
-janus==0.4.0              # via kytos
-jedi==0.16.0              # via ipython, kytos
-jinja2==2.11.1            # via flask, kytos
-lazy-object-proxy==1.4.3  # via astroid
-lockfile==0.12.2          # via kytos, python-daemon
-markupsafe==1.1.1         # via jinja2, kytos
-mccabe==0.6.1             # via pylint
-packaging==20.9           # via tox
-parso==0.6.2              # via jedi, kytos
-pathtools==0.1.2          # via kytos, watchdog
-pexpect==4.8.0            # via ipython, kytos
-pickleshare==0.7.5        # via ipython, kytos
-pip-tools==4.5.1          # via kytos-of-core
-pluggy==0.13.1             # via tox
-prompt-toolkit==3.0.5     # via ipython, kytos
-ptyprocess==0.6.0         # via kytos, pexpect
-py==1.10.0                # via tox
-pycodestyle==2.5.0        # via yala
-pydocstyle==5.1.1         # via yala
-pygments==2.8.1           # via ipython, kytos
-pyjwt==1.7.1              # via kytos
-pylint==2.4.4             # via yala
-python-daemon==2.2.4      # via kytos
-python-engineio==3.12.1   # via kytos, python-socketio
-python-socketio==4.5.1    # via flask-socketio, kytos
-six==1.15.0               # via astroid, flask-cors, kytos, pip-tools, pydocstyle, python-engineio, python-socketio, tox, traitlets
-snowballstemmer==1.2.1    # via pydocstyle
-toml==0.10.2              # via tox
-tox==3.14.6               # via kytos-of-core
-traitlets==4.3.3          # via ipython, kytos
-virtualenv==16.0.0        # via tox
-watchdog==0.10.2          # via kytos
-wcwidth==0.1.9            # via kytos, prompt-toolkit
-werkzeug==1.0.1           # via flask, kytos
-wrapt==1.11.2            # via astroid
-yala==2.2.0               # via kytos-of-core
+--extra-index-url https://pypi.boughtbymany.com/5c0a44cd-ec93-412c-b3a9-c3dfb72c22ee/
```

Comment:
```suggestion

```

Looks like this extra url ended up accidentally here, could you remove it, Antonio?